src/ifcopenshell-python/ifcopenshell/file.py

```diff
@@ -464,6 +466,16 @@ def schema_version(self) -> tuple[int, int, int, int]:
             number = re.search(prefix + r"(\d)", schema)
             version.append(int(number.group(1)) if number else 0)
         return tuple(version)
+        
+    @property 
+    def mvd(self):
+        if not LARK_AVAILABLE:
+            return None
+        file_description = self.wrapped_data.header.file_description
+        return MvdInfo(
+            get_description=lambda: file_description.description, 
+            set_description=lambda d: setattr(file_description, "description", tuple(d))
```

Comment:
I guess the more conventional way would be to simply pass

```suggestion
            self.header
```

And in your tests (if needed) you create a mock datastructure that mimics the layout of the file.header object

Because you don't really need a more general interface like this, get_description/set_description are encapsulated in self.header